src/components/MessageInput/__tests__/MessageInput.test.js

```diff
@@ -31,439 +36,440 @@ const t = (key) => key;
+      jest.clearAllMocks();
     });
-  }
 
-  const filename = 'some.txt';
-  const fileUploadUrl = 'http://www.getstream.io'; // real url, because ImagePreview will try to load the image
+    function dropFile(file, formElement) {
+      fireEvent.drop(formElement, {
+        dataTransfer: {
+          files: [file],
+        },
+      });
+    }
 
-  const getImage = () => new File(['content'], filename, { type: 'image/png' });
-  const getFile = () => new File(['content'], filename, { type: 'text/plain' });
+    const filename = 'some.txt';
+    const fileUploadUrl = 'http://www.getstream.io'; // real url, because ImagePreview will try to load the image
 
-  const mockUploadApi = () =>
-    jest.fn().mockImplementation(() =>
-      Promise.resolve({
-        file: fileUploadUrl,
-      }),
-    );
+    const getImage = () =>
+      new File(['content'], filename, { type: 'image/png' });
+    const getFile = () =>
+      new File(['content'], filename, { type: 'text/plain' });
+
+    const mockUploadApi = () =>
+      jest.fn().mockImplementation(() =>
+        Promise.resolve({
+          file: fileUploadUrl,
+        }),
+      );
 
-  const mockFaultyUploadApi = (cause) =>
-    jest.fn().mockImplementation(() => Promise.reject(cause));
+    const mockFaultyUploadApi = (cause) =>
+      jest.fn().mockImplementation(() => Promise.reject(cause));
 
-  it('Should shift focus to the textarea if the `focus` prop is true', async () => {
-    const { getByPlaceholderText } = renderComponent({
-      focus: true,
-    });
-    await waitFor(() => {
-      expect(getByPlaceholderText(inputPlaceholder)).toHaveFocus();
+    it('Should shift focus to the textarea if the `focus` prop is true', async () => {
+      const { getByPlaceholderText } = renderComponent({
+        focus: true,
+      });
+      await waitFor(() => {
+        expect(getByPlaceholderText(inputPlaceholder)).toHaveFocus();
+      });
     });
-  });
 
-  it('Should open the emoji picker after clicking the icon, and allow adding emojis to the message', async () => {
-    const {
-      container,
-      findByTitle,
-      queryByText,
-      getByDisplayValue,
-    } = renderComponent();
+    it('Should open the emoji picker after clicking the icon, and allow adding emojis to the message', async () => {
+      const {
+        container,
+        findByTitle,
+        queryByText,
+        queryAllByText,
+        getByDisplayValue,
+      } = renderComponent();
 
-    const emojiIcon = await findByTitle('Open emoji picker');
-    fireEvent.click(emojiIcon);
+      const emojiIcon = await findByTitle('Open emoji picker');
+      fireEvent.click(emojiIcon);
 
-    expect(queryByText('Pick your emoji…')).toBeInTheDocument();
+      expect(queryByText('Pick your emoji')).toBeInTheDocument();
 
-    const emoji = '💯';
-    const emojiButton = queryByText(emoji);
-    expect(emojiButton).toBeInTheDocument();
+      const emoji = '💯';
+      const emojiButton = queryAllByText(emoji)[0];
```

Comment:
Even after https://github.com/GetStream/stream-chat-react/pull/326/commits/9877314e03a2dd066edd7c564c4236fdad7edb0b#diff-4c02f7c9b62e8fcc93a2a781681a4b82L163, can we still have more than one of this emoji on the dom?

Reply:
Good point, forgot to change it back to what it was :)

Reply:
Oh, I see. Mocking localStorage just because of this seems a bit too much. I think it is fine to leave it as is. Or choose another emoji (I imagine this one is part of a default frequently used set 🤔 ).

Reply:
I actually picked this emoji because it isn't part of the standard frequently used set :P That worked fine when I ran all of these tests once, but now that they're repeated for all UI components the localstorage thing is messing things up again :)